content/en/docs/tasks/manage-kubernetes-objects/update-api-object-kubectl-patch.md

```diff
@@ -445,6 +445,7 @@ and
 [kubectl apply](/docs/reference/generated/kubectl/kubectl-commands/#apply).
 
 
+Notice that strategic merge patch is unsupported for Custom Resource objects.
```

Comment:
This could be:
```suggestion
{{< note >}}
Strategic merge patch is not supported for {{< glossary_tooltip term_id="CustomResourceDefinition" text="custom resources" >}}.
{{< /note >}}
```

@pacoxu, what do you think?